tests/test_scala.py

```diff
@@ -0,0 +1,29 @@
+def test_java_exists(Command):
+	java = Command("java -version")
+	assert java.rc == 0
+
+
+def test_scala_exists(Command, Ansible, SystemInfo):
+	scala_version = Command(
+		"scala -e \"println(util.Properties.versionNumberString)\"")
+	assert scala_version.rc == 0
+
+def test_scala_version(Command, Sudo):
+	with Sudo(user='root'):
+		# Extract target scala version from ansible facts
+		target_scala_version = Command.check_output(
```

Comment:
I was initially trying to solve the problem of handling version overrides, but that's ultimately not important in these tests. Fixed in [`9d58d93`](https://github.com/azavea/ansible-scala/pull/3/commits/9d58d93e59721c3a5288cf468e367ceef69ca8e4)